api/server.py

```diff
@@ -26,31 +26,58 @@ def run(port, hostname, debug_mode):
     app.run(host=hostname, port=port, debug=debug_mode)
 
 
-# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
+# TODO: If functionality "meta-paths for node set A and B" will be written in Java, team alpha will need this information in Java
 @app.route("/node-sets", methods=["POST"])
 def receive_node_sets():
+    """
+    Endpoint where the node sets from the "Setup" page which the user selects are posted.
+    This endpoint is called for each new added node.
+
+    The repeated calling enables us to start the following computations as early as possible
+    so that we can return information on the next pages faster.
+    For example on the first call we already know the type of the whole node set and
+    therefore can begin to retrieve the corresponding node sets.
+    """
     # TODO: Check if necessary information is in request object
     if not request.json:
         abort(400)
+    raise NotImplementedError("This API endpoint isn't implemented in the moment")
 
 
 @app.route("/node-sets", methods=["GET"])
 def send_node_sets():
+    """
+    Endpoint where the node sets which the user previously selected on the "Setup" page can be retrieved.
+    """
+    # TODO: Does active_learning really needs this endpoint? Does someone needs this endpoint?
     # TODO: Call fitting method in active_learning
     # TODO: Check if necessary information is in request object
+    raise NotImplementedError("This API endpoint isn't implemented in the moment")
     return jsonify("Hello world")
 
 
-# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
+# TODO: If functionality "meta-paths for node set A and B" will be written in Java, team alpha will need this information in Java
 @app.route("/types", methods=["POST"])
 def receive_edge_node_types():
+    """
+    Endpoint where the node and edge types which are selected (types which are active) on the "Config" page are posted
+    """
     # TODO: Check if necessary information is in request object
     if not request.json:
         abort(400)
+    raise NotImplementedError("This API endpoint isn't implemented in the moment")
 
 mock_id = 1
 @app.route("/next-meta-paths", methods=["GET"])
 def send_next_metapaths_to_rate():
+    """
+    Endpoint which returns the next `batchsize` meta-paths to rate.
```

Comment:
Maybe specify the parameters this route requires? E.g. Batch-size

Reply:
In the moment `batchsize` is a hard coded variable and not a parameter. So this would also be something for an issue or another PR/branch.

Reply:
Mentioned on #28